### DISCUSSION TITLE
Is it possible to nest Copier Templates?

### DISCUSSION THREAD

**Discussion by devinnasar** [copier-org/copier#1788] [General]
Hello,

I'm thinking about Copier for a project where I need to manage the structure of a repo and deploy multiple templates as subfolders of this repo. My goal is to be able to run `copier update` against the root of the repo and have Copier update both the root project and any of the subfolder projects which it detects are also Copier templates. I couldn't find this scenario described in the docs. Is this something that's supported, or would one have to wrap the call to `copier update` in a script of their own? Thanks!

**Comment by yajo** [comment]
Yes, you will probably have to do your own scripting for this.
